broker-daemon/block-order-worker/index.js

```diff
@@ -241,8 +241,8 @@ class BlockOrderWorker extends EventEmitter {
    * @throws {Error} If there are insufficient outbound or inbound funds
    */
   async checkFundsAreSufficient (blockOrder) {
-    const { marketName, side, outboundSymbol, inboundSymbol } = blockOrder
-    const { activeOutboundAmount, activeInboundAmount } = await this.calculateActiveFunds(marketName, side)
+    const { outboundSymbol, inboundSymbol } = blockOrder
+    const { activeOutboundAmount, activeInboundAmount } = await this.calculateActiveFunds()
```

Comment:
seems like we should still be passing the side here, right?

Reply:
yeah we need the side here